### PR TITLE
Reduce allocations in MinDisspLen

### PR DESCRIPTION
**Pull request by charleskawczynski**
I'm hoping that this will supersede #546.

### CODE REVIEW

**Comment by yairchn** [comment]
there is some weird issue in dycoms - I will try to look better tomorrow

File: src/update_aux.jl [hunk 1]
```diff
@@ -336,6 +336,22 @@ function update_aux!(edmf, gm, grid, state, Case, param_set, TS)
     ##### compute_eddy_diffusivities_tke
     #####
 
+    # Subdomain exchange term
+    Ic = CCO.InterpolateF2C()
+    b_exch = center_aux_turbconv(state).b_exch
+    a_en = aux_en.area
+    w_en = aux_en_f.w
+    tke_en = aux_en.tke
+    for i in 1:(up.n_updrafts)
```
Comment (yairchn) [inline src/update_aux.jl:345]: shouldn't b_exch be set to zero before the loop over updrafts ?

Comment (yairchn) [reply src/update_aux.jl:345]: adding '@. b_exch .*= 0' in line 343 fixed the problem

Comment (charleskawczynski) [reply src/update_aux.jl:345]: Yeah, good catch!

**Review by yairchn — APPROVED** [review]
LGTM - some table updates are needed but the results are qualitatively unchanged

**Comment by charleskawczynski** [comment]
Hmm, not sure what's going on here, this PR actually _hurt_ performance, despite resulting in far less allocations per tendencies evaluations. We may want to review this or look into it further.

**Pull request merged**